codec-http2/src/main/java/io/netty/handler/codec/http2/ReadOnlyHttp2Headers.java

```diff
@@ -769,6 +771,30 @@ public CharSequence status() {
         return get(PseudoHeaderName.STATUS.value());
     }
 
+    @Override
+    public boolean contains(CharSequence name, CharSequence value, boolean caseInsensitive) {
+        final int nameHash = AsciiString.hashCode(name);
+        final HashingStrategy<CharSequence> strategy = caseInsensitive? CASE_INSENSITIVE_HASHER : CASE_SENSITIVE_HASHER;
+        final int valueHash = strategy.hashCode(value);
+
+        return contains(name, nameHash, value, valueHash, strategy, pseudoHeaders)
+               || contains(name, nameHash, value, valueHash, strategy, otherHeaders);
+    }
+
+    private static boolean contains(CharSequence name, int nameHash, CharSequence value, int valueHash,
```

Comment:
can we use this for `contains(CharSequence name, CharSequence value)`? looks like there is a bug in the value comparison which should be case sensitive.

Reply:
followup PR is fine for this

Reply:
@Scottmitch I think that was what @tsegismont wanted to do one this is merged: https://github.com/netty/netty/pull/7633#issuecomment-360745218

Reply:
@Scottmitch yep, see my other comments in the PR, I noticed the bug and Norman asked to fix it separately